webapp/tests/karma/ts/modules/contacts/contacts-content.component.spec.ts

```diff
@@ -80,9 +80,9 @@ describe('Contacts content component', () => {
     };
 
     selectedContact = {
```

Comment:
Removing the default values from selectedContact to make test easier to understand as suggested. Then below some test cases are overriding the selector with these values.